azure-pipelines/signing.yml

```diff
@@ -496,14 +496,13 @@ extends:
             solution: 'azure-pipelines\nuget-package.signproj'
             msbuildArguments: '/p:OutDir=$(Build.ArtifactStagingDirectory)\vs-insertion\drop /p:IntermediateOutputPath=$(Build.ArtifactStagingDirectory)\vs-insertion\drop'
         - task: APIScan@2
-          env:
-            AzureServicesAuthConnectionString: RunAs=App;AppId=d318cba7-db4d-4fb3-99e1-01879cb74e91
           inputs:
             softwareFolder: '$(Build.ArtifactStagingDirectory)\drop'
             softwareName: 'vcpkg'
             softwareVersionNum: '$(VCPKG_BASE_VERSION)'
             softwareBuildNum: '$(Build.BuildId)'
             symbolsFolder: '$(Build.ArtifactStagingDirectory)\drop'
+            azureSubscription: 'VSEng-APIScanSC'
```

Comment:
I am starting to hear reports that we may need to do something more than just add azureSubscription.  Which is odd because my testing did not need it.  Are you able to test this before checking in?  Thx.